test/fixtures/trailing-slash/trailish-slash.test.js

```diff
@@ -0,0 +1,5 @@
+import { buildFixture } from '../../utils/build'
```

Comment:
should be trailing-slash.test.js (?)

Reply:
Nice fine! trailish is seems likely name for current status 😆